app/controllers/message_handler_controller.rb

```diff
@@ -0,0 +1,5 @@
+class MessageHandlerController < ApplicationController
+  def handle
+
```

Comment:
am I missing something here? how do requests start interacting with the AkCreator classes if this method is blank?